tools/create_virtual_hardware.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Make me a Gimlet!
+#
+# The entire control plane stack is designed to run and operate on the Oxide
+# rack, in each Gimlet. But Gimlet's don't quite exist yet. In the meantime,
+# this script can be used to create a few pieces of virtual hardware that
+# _simulate_ a Gimlet, allowing us to develop software that approximates the
+# eventual operation on Oxide hardware.
+#
+# See `docs/virtual-hardware.adoc` for more details.
```

Comment:
This doc doesn't exist; `how-to-run.adoc` instead?

Reply:
Thanks, meant to add some details about why this is all needed in that file. Will add.